client/src/containers/Corruption/CorruptionView.tsx

```diff
@@ -10,10 +10,17 @@ type CorruptionViewProps = {
   corruption;
 };
 
+const CORRUPTION_INDEX_IDS = {
+  "CORRUPTION_RATING": "0",
```

Comment:
You don't need additional `CORRUPTION_` pefix. Just leave
const CORRUPTION_INDEXES {
RATING: "",
POSITION: ""}

even bettre 
enum INDICES {
SCORE,
RANK
}